jarbas/api/tests/test_reimbursement_view.py

```diff
@@ -1,59 +1,19 @@
-from datetime import date
 from json import loads
 from unittest.mock import patch
+from urllib.parse import urlencode
 
-from django.core.cache import cache
 from django.shortcuts import resolve_url
 from django.test import TestCase
 
+from mixer.backend.django import mixer
+
 from jarbas.core.models import Reimbursement
-from jarbas.core.tests import sample_reimbursement_data, suspicions
 
 
 class TestListApi(TestCase):
 
     def setUp(self):
-
-        data = [
-            sample_reimbursement_data.copy(),
-            sample_reimbursement_data.copy(),
-            sample_reimbursement_data.copy(),
-            sample_reimbursement_data.copy(),
-            sample_reimbursement_data.copy()
-        ]
-
-        data[1]['cnpj_cpf'] = '22222222222'
-        data[1]['document_id'] = 42 * 2
-        data[1]['issue_date'] = date(1969, 12, 31)
-        data[1]['probability'] = None
-        data[1]['subquota_id'] = 22
-
-        data[2]['applicant_id'] = 13 * 3
-        data[2]['cnpj_cpf'] = '22222222222'
-        data[2]['document_id'] = 42 * 3
-        data[2]['probability'] = 0.1
-        data[2]['subquota_id'] = 22
-
-        data[3]['applicant_id'] = 13 * 4
-        data[3]['cnpj_cpf'] = '22222222222'
-        data[3]['document_id'] = 42 * 4
-        data[3]['probability'] = 0.9
-        data[3]['subquota_id'] = 22
-        data[3]['year'] = 1983
-        data[3]['issue_date'] = '1970-02-01'
-
-        data[4]['applicant_id'] = 13 * 5
-        data[4]['cnpj_cpf'] = '22222222222'
-        data[4]['document_id'] = 42 * 5
-        data[4]['subquota_id'] = 22
-        data[4]['year'] = 1983
-        data[4]['issue_date'] = '1960-02-01'
-        del data[4]['probability']
-        del data[4]['suspicions']
-
-        for fixture in data:
-            Reimbursement.objects.create(**fixture)
-
+        mixer.cycle(5).blend(Reimbursement)
```

Comment:
Actually the 5 reimbursements created before had controlled differences to make the tests more precise. Surely the previous code was poor and yours is way better. So… just to confirm: `5` here is quite arbitrary (before we need five due to the combination of differences to properly test API requests), right? I mean… when we need actually with a specificity now we create on the test itself, not during `setUp`, is that right? If so we might reduce from 5 to… whatever, 3. What do you think?